app/controllers/login_controller.rb

```diff
@@ -7,11 +7,15 @@ def logout
   def public
     request.env['warden'].logout if github_authenticated? :private
     github_authenticate! :default
+    @user = gh.user
+    @emails = gh.user.emails
     redirect_to params[:redirect_to] || "/"
   end
   def private
     request.env['warden'].logout if github_authenticated? :default
     github_authenticate! :private
+    @user = gh.user
+    @emails = gh.user.emails
```

Comment:
I'm afraid this might throw a ghee error causing a redirect to an error page

Please may sure and verify that is not the case. 

We will likely have a transition period where we do not have the email scope